.circleci/config.yml

```diff
@@ -1854,7 +1854,6 @@ jobs:
           executor: <<pipeline.parameters.machine-image>>
           extras-requires: anchorecli
       - run: sudo snap install yq
```

Comment:
jq is definitely used in the script. I'd suggest changing line 1856 to `sudo snap install yq && sudo snap install jq`?